components/volume/types/interface.go

```diff
@@ -0,0 +1,26 @@
+package types
+
+import (
+	// TODO: the interface should only expose types from this package
+	"github.com/docker/docker/volume"
+)
+
+const (
+	// ComponentType is the name identifying this type of component
+	ComponentType = "volumes"
```

Comment:
So what is the point of a component type? Why have components if they are just singletons?